lms/djangoapps/discussion/django_comment_client/base/tests.py

```diff
@@ -1962,7 +1962,7 @@ def _raise_navigation_event(self, label, include_name):
             data=self.create_segmentio_event_json(**kwargs),
             content_type='application/json',
         )
-        User.objects.create(pk=SEGMENTIO_TEST_USER_ID, username=str(mock.sentinel.username))
+        User.objects.create(username=str(mock.sentinel.username))
```

Comment:
Won't this create a user with random PK? If some tests are depending on that specific pk then it would fail.

Reply:
it does but it doesn't affect any test

Reply:
with `pk` provided, it raises User Not found